README.md

```diff
@@ -216,7 +216,67 @@ The possible values are
 
 The `--vcs` value will be passed as is to cargo.
 
+### Subprocesses
+
+You can spawn subprocess along your temporary shell like this:
+
+```toml
+[[subprocess]]
+command = "alacritty -e cargo-watch -x run"
+foreground = false
+```
+
+The `command` field is a shell command like `echo Hello`.
+The `foreground` field allows to run the program in foreground instead of
+background.
+
+#### Additional settings
+
+* `working_dir` overrides the default working directory which is the temporary
+  directory.
+* `keep_on_exit` is used to know if the process will be killed when exiting the
+  shell or not. The default will be false and this setting doesn't work with
+  foreground process.
```

Comment:
Invalid use of "and" here. There must be a relation between the elements to use "and" like that.

```suggestion
* `keep_on_exit` is used to keep the process alive after exiting the
  shell. The default is to kill the process when the shell exits.
  This setting doesn't work with foreground process.
```